loadbalancer/src/main/java/com/io/loadbalancer/consistent_hashing/Hashing.java

```diff
@@ -0,0 +1,46 @@
+package com.io.loadbalancer.consistent_hashing;
+
+import com.io.loadbalancer.exceptions.NoIMInstanceAvailableException;
+
+import java.util.Arrays;
+import java.util.Random;
+
+public class Hashing {
```

Comment:
Can't this class be responsible for returning `ImageManager` instance instead of an index to a proper list of image managers?

Reply:
Done